core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -2220,6 +2223,17 @@ public ServerBuilder http1HeaderNaming(Http1HeaderNaming http1HeaderNaming) {
         return this;
     }
 
+    /**
+     * Adds additional allowed HTTP methods to the default set of allowed {@link HttpMethod}s.
+     *
+     * @param additionalAllowedHttpMethods the additional allowed HTTP methods
+     */
+    public ServerBuilder additionalAllowedHttpMethods(String... additionalAllowedHttpMethods) {
```

Comment:
optional) for consistency with other APIs, it may be worth adding a `additionalAllowedHttpMethods(Iterable<String>)` variant